s2i/config/krb5.conf

```
@@ -1,30 +1,35 @@
-# To opt out of the system crypto-policies configuration of krb5, remove the
-# symlink at /etc/krb5.conf.d/crypto-policies which will not be recreated.
 includedir /etc/krb5.conf.d/
 
-[logging]
-    default = FILE:/var/log/krb5libs.log
-    kdc = FILE:/var/log/krb5kdc.log
-    admin_server = FILE:/var/log/kadmind.log
+# depending on your config, you may wish to uncomment the following:
+# includedir /var/lib/sss/pubconf/krb5.include.d/
 
 [libdefaults]
-    dns_lookup_realm = false
-    ticket_lifetime = 24h
-    renew_lifetime = 7d
-    forwardable = true
-    rdns = false
-    pkinit_anchors = FILE:/etc/pki/tls/certs/ca-bundle.crt
-    spake_preauth_groups = edwards25519
-    default_realm = REDHAT.COM
-    dns_canonicalize_hostname = fallback
+  default_realm = IPA.REDHAT.COM
+  dns_lookup_realm = true
+  dns_lookup_kdc = true
+  rdns = false
+  dns_canonicalize_hostname = false
+  ticket_lifetime = 24h
+  forwardable = true
+  udp_preference_limit = 0
 
 [realms]
- REDHAT.COM = {
-   kdc = kerberos.corp.redhat.com:88
-   admin_server = kerberos.corp.redhat.com:749
-   default_domain = redhat.com
- }
 
-[domain_realm]
- .redhat.com = REDHAT.COM
- redhat.com = REDHAT.COM
+    REDHAT.COM = {
+        default_domain = redhat.com
+        dns_lookup_kdc = true
+        master_kdc = kerberos.corp.redhat.com
+        admin_server = kerberos.corp.redhat.com
+    }
+
+    #make sure to save the IPA CA cert
+    #mkdir /etc/ipa && curl -o /etc/ipa/ca.crt https://password.corp.redhat.com/ipa.crt
+    IPA.REDHAT.COM = {
+       pkinit_anchors = FILE:/etc/ipa/ca.crt
+       pkinit_pool = FILE:/etc/ipa/ca.crt
+       default_domain = ipa.redhat.com
+       dns_lookup_kdc = true
+       # Trust tickets issued by legacy realm on this host
+       auth_to_local = RULE:[1:$1@$0](.*@REDHAT\.COM)s/@.*//
+       auth_to_local = DEFAULT
+    }
```

```suggestion
    }

```

can we please add an empty end line, to respect the standard